### DISCUSSION TITLE
Tips for Windows setup

### DISCUSSION THREAD

**Discussion by ghost** [itsToggle/plex_debrid#683] [General]
1. Download and install [Microsoft Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/) if you're getting a `could not build wheels for regex, which is required to install py project.toml-based projects` error when running `pip install -r requirements.txt`
2. If you're getting a ` --vfs-cache-mode writes or full is recommended for this remote as it can't stream` error when running rclone, just add `--vfs-cache-mode writes` to the end of the command like so:
                `./rclone.exe cmount realdebrid: X: --dir-cache-time 10s --vfs-cache-mode writes`

That's it. Those are my 2 issues that I ran into when following the Windows installer instructions that weren't actually noted in the guidance.  

I'm running Windows 11 Home edition. Nothing fancy.

**Comment by JansoD95** [comment]
Thank you so much, was absolutely stumped by the first part.
